build.gradle.kts

```diff
@@ -21,6 +22,16 @@ infra {
     }
 }
 
+apiValidation {
+    ignoredPackages += listOf(
+        "kotlinx.collections.immutable.internal"
```

Comment:
The package doesn't seem to have any public declaration currently. But I think it is important to catch unintended public declarations in this package in the future.

Reply:
Done.